### ISSUE TITLE
logging moderation actions 

### ISSUE THREAD

**Issue by kmahar** [haystack/murmur#158]
Was looking at the Email Valet paper and they talk about how they had a log basically that the user could look at to see which of their messages had been opened/viewed by the assistant. 
I think later on that would be a nice feature to have, where we basically keep a record of who looks at and approves/rejects/etc. each message that comes through. This might help more private people feel better about someone else seeing their emails. And maybe would also increase their willingness to have paid strangers or volunteers be in their squad. 
For the first version we can just put a message in the footer of the email saying "this message was approved by so-and-so".

**Comment by karger** [comment]
it could also help people decide if certain moderators weren't doing a 
good job.  maybe just to talk to them.  Also, I hope we have a way for a 
user to *remove* a moderator if they think they are not doing a good job.

**Comment by kmahar** [comment]
yes, it's possible to change user's member roles and also to delete them from a group/squad.
